src/PythonMigrationViewExtension/PythonMigrationAssistantViewModel.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Dynamo.PythonMigration.MigrationAssistant;
+using PythonNodeModels;
+using System.Collections.Generic;
+
+namespace Dynamo.PythonMigration
+{
+    internal class PythonMigrationAssistantViewModel
+    {
+        public string OldCode { get; set; }
+        public string NewCode { get; set; }
+
+        public PythonMigrationAssistantViewModel(PythonNode pythonNode)
+        {
+            OldCode = pythonNode.Script;
+            MigrateCode();
+        }
+
+        private void MigrateCode()
+        {
+            var migrationInputs = new List<string>()
+            {
+                "code"
+            };
```

Comment:
we might need better names for this var/param, don't understand how "code" is an input ?

Reply:
yea it is a bit confusing, but i think thats more the whole using Python.net way thats confusing. The reason that "code" is an input in this case, is that my python script has one variable `output` which calls the method `transform` which requires 1 argument which is the `code`. "code" is what needs to be migrated, could change it to `script` if thats better.